oci/oci.go

```diff
@@ -515,7 +515,9 @@ func (r *Runtime) StopContainer(c *Container, timeout int64) error {
 	}
 	done := make(chan struct{})
 	go func() {
-		for {
+		ticker := time.NewTicker(time.Duration(timeout) * time.Second)
```

Comment:
alrighty, I'll just add a 100ms delay after first check and continue the loop then?
```
for {
    exited = exit container call
    if exited {
       break
    }
    delay 100ms
}
```